docs/docsource/operation/platform_operations/retrieve_access_tokens.rst

```diff
@@ -6,46 +6,51 @@ Retrieve Access Tokens
 Prerequisites
 ~~~~~~~~~~~~~
 
-- jq and curl installed
-- Access from the browser user machine
+-kubectl, jq, and curl installed
 
 Procedure
 ~~~~~~~~~
 
-#. Retrieve ``ACCESS_TOKEN``, ``TOKEN``, and ``INSTANCE_ID``:
+.. note::
 
-   .. code::
-      
-      OIDC_REALM='zenko-realm'
+   In the following commands, replace ``<username>`` and ``<password>`` with
+   the user credentials provisioned for the OIDC service, and ``<zenko-name>``
+   with the name of the deployed Zenko instance.
```

Comment:
Consider putting a :ref: to where/how the the user can provision the appropriate credentials.
operation/account_operations/index.rst:       `Account Operations` ?

Reply:
the OIDC credentials would be different to the account described in `operation/account_operations/index.rst`.